### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by ssy352959096**

### CODE REVIEW

File: README_zh.md [hunk 1]
```diff
@@ -105,3 +105,19 @@ profile.setHttpClientConfig(clientConfig);
 IAcsClient client = new DefaultAcsClient(profile);
 ```
 
+## 超时机制
+Request 设置 -> Client 设置 -> 默认，优先级依次降低；
+默认连接超时为5秒，读超时为10秒；
+```java
+//Client超时设置，对当前所有request有效，优先级高于默认
```
Comment (sjj3086786) [inline README_zh.md]: //Client超时设置，对当前所有request有效，优先级高于默认，低于Request

File: README.md [hunk 2]
```diff
@@ -116,4 +116,21 @@ clientConfig.setMaxRequests(60);
 
 profile.setHttpClientConfig(clientConfig);
 IAcsClient client = new DefaultAcsClient(profile);
+```
+
+## Timeout
+Request Settings -> Customer Settings -> Default, priority from high to low;
```
Comment (sjj3086786) [inline README.md]: Customer Settings --> Client Settings

Comment (sjj3086786) [reply README.md]: Default --> Default Settings

File: README.md [hunk 3]
```diff
@@ -116,4 +116,21 @@ clientConfig.setMaxRequests(60);
 
 profile.setHttpClientConfig(clientConfig);
 IAcsClient client = new DefaultAcsClient(profile);
+```
+
+## Timeout
+Request Settings -> Customer Settings -> Default, priority from high to low;
+The default ConnectTimeout is 5 seconds and the ReadTimeout is 10 seconds.
+```java
+//The Client timeout setting is valid for all current requests, with priority higher than the default
```
Comment (sjj3086786) [inline README.md]: The Client timeout settings are valid for all requests

Comment (sjj3086786) [inline README.md]: The timeout setting of client is valid for all requests

File: README.md [hunk 4]
```diff
@@ -116,4 +116,21 @@ clientConfig.setMaxRequests(60);
...
+```java
+//The Client timeout setting is valid for all current requests, with priority higher than the default
+HttpClientConfig clientConfig = HttpClientConfig.getDefault();
+clientConfig.setReadTimeoutMillis(readTimeoutMillis);
+clientConfig.setConnectionTimeoutMillis(connectionTimeoutMillis);
+IClientProfile profile = DefaultProfile.getProfile(regionId, accesskeyId, accesskeySecret);
+profile.setHttpClientConfig(clientConfig);
+DefaultAcsClient client = new DefaultAcsClient(profile);
+
+// Request timeout setting, valid only for current request, highest priority
```
Comment (sjj3086786) [inline README.md]: valid only for current request, highest priority --> valid only for the current request, has the highest priority

File: README.md (hunk 3 above)
Comment (ssy352959096) [reply README.md]: The Client Setting is valid for all requests, with priority higher than default and lower than request

File: README.md [hunk 5]
```diff
@@ -116,4 +116,21 @@ clientConfig.setMaxRequests(60);
 
 profile.setHttpClientConfig(clientConfig);
 IAcsClient client = new DefaultAcsClient(profile);
+```
+
+## Timeout
+Request Settings -> Client Settings -> Default Settings, priority from high to low;
+The default ConnectTimeout is 5 seconds and the ReadTimeout is 10 seconds.
+```java
+//The client setting is valid for all requests, with priority higher than the default and lower than request
```
Comment (sjj3086786) [inline README.md]: ”, with priority higher than the default and lower than request“
这句去掉吧，优先级前面定义过了，语法上读起来也有点问题

**Pull request merged**